d2l-scroll-wrapper.html

```diff
@@ -125,6 +125,7 @@
 
 			attached: function() {
 				this.async(this.checkScrollbar);
+				this.async(this.checkScrollbar);
```

Comment:
All browsers will be affected by this change right? Does it have any perf impact?